js/src/carousel.js

```diff
@@ -492,6 +503,9 @@ class Carousel {
       $(this._element).trigger(slidEvent)
     }
 
+    // does not wait for the transition to complete
+    this._activeElement = nextElement
```

Comment:
Can you explain this change please?

Reply:
this PR is currently out of data (so I reverted it to draft). See https://github.com/twbs/bootstrap/pull/31818